hawc/apps/eco/migrations/0003_auto_20220719_1256.py

```diff
@@ -0,0 +1,193 @@
+# Generated by Django 3.2.12 on 2022-07-19 16:56
```

Comment:
squash eco migrations? i think there just needs to be 2 migrations - one that creates all the models, and one that loads the data